packages/babel-preset-udemy-website/tests/result/css-modules.js

```diff
@@ -1,46 +1,41 @@
-'use strict';
+"use strict";
 
-import _getClassName from 'babel-plugin-react-css-modules/dist/browser/getClassName';
+import _getClassName from "babel-plugin-react-css-modules/dist/browser/getClassName";
+import '../babel-polyfill';
 import React, { Component } from 'react';
-
 import baseStyles from './css-modules.global.less';
 import styles from './css-modules.less';
-
 var _styleModuleImportMap = {
-    'styles': {
-        'greeting': 'css-modules--greeting--3-JKt'
-    }
+  "styles": {
```

Comment:
There were a bunch of stylistic things that changed in how Babel generates things.